contracts/multiswap-contracts/FiberRouter.sol

```diff
@@ -79,7 +77,7 @@ contract FiberRouter is Ownable, TokenReceivable {
         bytes multiSignature
     );
 
-    event RouterAndSelectorWhitelisted(address router, bytes selector);
+    event RouterAndSelectorWhitelisted(address router, bytes4 selector);
     event RouterAndSelectorRemoved(address router, bytes selector);
```

Comment:
byte4? for consistency

Reply:
Yes right, as selectors are commonly represented as bytes4 in Solidity.

Reply:
So make RouterAndSelectorRemoved byte4 too